evalml/model_understanding/prediction_explanations/_user_interface.py

```diff
@@ -166,86 +169,100 @@ def make_drill_down_dict(provenance, shap_values, normalized_values, pipeline_fe
 
     @abc.abstractmethod
     def make_text(self, aggregated_shap_values, aggregated_normalized_values,
-                  shap_values, normalized_values, pipeline_features, original_features):
+                  shap_values, normalized_values, pipeline_features, original_features, expected_value):
         """Creates a table given shap values and formats it as text."""
 
     @abc.abstractmethod
     def make_dict(self, aggregated_shap_values, aggregated_normalized_values,
-                  shap_values, normalized_values, pipeline_features, original_features):
+                  shap_values, normalized_values, pipeline_features, original_features, expected_value):
         """Creates a table given shap values and formats it as dictionary."""
 
     def make_dataframe(self, aggregated_shap_values, aggregated_normalized_values,
-                       shap_values, normalized_values, pipeline_features, original_features):
+                       shap_values, normalized_values, pipeline_features, original_features, expected_value):
         data = self.make_dict(aggregated_shap_values, aggregated_normalized_values,
                               shap_values=shap_values, normalized_values=normalized_values,
-                              pipeline_features=pipeline_features, original_features=original_features)['explanations']
+                              pipeline_features=pipeline_features, original_features=original_features,
+                              expected_value=expected_value)['explanations']
 
         # Not including the drill down dict for dataframes
         # 'drill_down' is always included in the dict output so we can delete it
         for d in data:
             del d['drill_down']
 
+        if not isinstance(data, list):
+            data = [data]
```

Comment:
I came across an instance of the data being passed through as a list, like it seems to expect.